src/DbInterface.cpp

```diff
@@ -817,6 +818,54 @@ void DbInterface::getSoCIpAddress(std::shared_ptr<swss::DBConnector> configDbCon
     processSoCIpAddress(entries);
 }
 
+// ---> warmRestartReconciliation(const std::string &portName);
+//
+// port warm restart reconciliation procedure
+//
+void DbInterface::warmRestartReconciliation(const std::string &portName)
+{
+    MUXLOGDEBUG(portName);
+
+    if (isWarmStart()) {
+        setMuxMode(portName, "auto");
```

Comment:
Note that per-port setting muxmode as `auto` can temporarily cause a subset of ports to be in auto, and others in manual mode.
I don't foresee any issues with that (do you?), but I am curious why you don't wait for all ports to reconcile (statemachine init done), and set all the ports together as `auto`? Also, that way setting them as auto would be consistent with setting them as manual (all at once).